chat/client/src/components/Chat/Chat.js

```diff
@@ -1,68 +1,103 @@
-import React, { useState, useEffect } from "react";
+import React, { useState, useEffect } from 'react';
 import queryString from 'query-string';
-import io from "socket.io-client";
+import io from 'socket.io-client';
+import './Chat.css';
 
-import TextContainer from '../TextContainer/TextContainer';
-import Messages from '../Messages/Messages';
-import InfoBar from '../InfoBar/InfoBar';
-import Input from '../Input/Input';
+import Infobar from '../infobar/infobar.js';
+import Input from '../Input/Input.js';
+import Messages from '../Messages/Messages.js';
+let socket;  //используем io
 
-import './Chat.css';
 
-const ENDPOINT = 'https://project-chat-application.herokuapp.com/';
+//СДЕЛАТЬ CSS (/chat/chat.css)
+//import './Chat.css';
+
+const Chat = ({location}) => {
+    const [name, setName] = useState('');
+    const [room, setRoom] = useState('');
+    const [message, setMessage] = useState('');
+    const [messages, setMessages] = useState([]);  
+    const ENDPOINT = 'http://localhost:5000'; //'localhost:5000'  '5000
```

Comment:
What if your backend will have different URL?

You can define variables in webpack during build https://stackoverflow.com/a/30061249